### DISCUSSION TITLE
ld-select

### DISCUSSION THREAD

**Discussion by ArpithaKamath** [emdgroup-liquid/liquid#477] [General]
Can ld-select be modified to creatable dropdown such that when you search for an option.
Example: 
![image](https://user-images.githubusercontent.com/60575192/213479286-db321ae5-7b80-486f-bd35-7470830d1960.png)

**Comment by borisdiakur** [comment]
Hi @ArpithaKamath 
There is an open feature request for exactly that: https://github.com/emdgroup-liquid/liquid/issues/463
Can you please upvote the issue with 👍 on the [issue description](https://github.com/emdgroup-liquid/liquid/issues/463#issue-1527542170). We'll put this on our road map.

**Comment by borisdiakur** [comment]
Hi @ArpithaKamath, thanks again for clarifying the requirement and creating the [feature request for a "creatable select"](https://github.com/emdgroup-liquid/liquid/issues/478). 🙏
